src/pages/api/thirdparty/graphql.tsx

```diff
@@ -0,0 +1,87 @@
+import { ApolloServer } from '@apollo/server'
+import { DateTime } from 'luxon'
+import { startServerAndCreateNextHandler } from '@as-integrations/next'
+import { GraphQLJSON } from 'graphql-type-json'
+import { GraphQLError } from 'graphql'
+import KeystoneAPI from '../dataSources/keystone'
+
+/* Type Definitions */
+
+/* Data we will return to any third-party querying the API. */
+const typeDefs = `
+  scalar JSON
+
+  type Article {
```

Comment:
TODO: The data we return may need to be updated based on requirements from third-party